### PR TITLE
[SDPA-1240] Added external link icon to RplRelatedLinks and RplWhatsN…

### PR DESCRIPTION
**Pull request by dylankelly**
**JIRA issue:** https://digital-engagement.atlassian.net/browse/SDPA-1240

### Changed

1.  Added arrow_right_primary icon to RplRelatedLinks
1.  Added arrow_right_primary icon to RplWhatsNext
1. Updated demo data to include external url

### Screenshots

screen shot 2018-11-08 at 1 21 25 pm

### CODE REVIEW

File: packages/Molecules/RelatedLinks/index.vue [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <div class="rpl-related-links__row">
       <ul class="rpl-related-links__items" v-if="links">
         <li class="rpl-related-links__item" v-for="(item, index) of links" :key="index">
-          <rpl-text-link :url="item.url" :text="item.text" :underline="true" size="small" theme="dark" />
+          <rpl-text-link :url="item.url" icon-symbol="arrow_right_primary" icon-color="white" icon-size="small" :text="item.text" :underline="true" size="small" theme="dark" />
```
Comment (tim-yao) [inline packages/Molecules/RelatedLinks/index.vue]: Hey @dylankelly The `icon-size` options are here: https://github.com/dpc-sdp/ripple/blob/master/packages/Atoms/Icon/Icon.vue#L38
```
sizes: {
        's': 0.5,
        'm': 1,
        'l': 1.5,
        'xl': 2,
        'xxl': 2.5
      }
```
Can `s`, `m`... or any number, so value `small` won't work here I think.

File: packages/Molecules/WhatsNext/index.vue [hunk 2]
```diff
@@ -6,7 +6,7 @@
     <div class="rpl-whats-next__row">
       <ul class="rpl-whats-next__items" v-if="links">
         <li class="rpl-whats-next__item" v-for="(item, index) of links" :key="index">
-          <rpl-text-link :url="item.url" :text="item.text" :underline="true" size="small" />
+          <rpl-text-link :url="item.url" :text="item.text" icon-symbol="arrow_right_primary" icon-size="small" :underline="true" size="small" />
```
Comment (tim-yao) [inline packages/Molecules/WhatsNext/index.vue]: Same `icon-size` issue as above.

Comment (dylankelly) [reply packages/Molecules/WhatsNext/index.vue]: ah yes, must have got mixed up with the size prop which takes 'small'. Might be a good idea to standardize constant units. I've updated the PR.

**Comment by tim-yao** [comment]
Hey @dylankelly need to update the snapshot test as the story changed.

**Comment by dylankelly** [comment]
@tim-yao Snapshot updated and branch up to date

**Pull request merged**